main.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2016 Table Top Inventing
+# All rights reserved.
+
+import cgi
+import json
+import urllib
+import webapp2
+from google.appengine.ext import ndb
+
+
+class measurePair(ndb.Model):
+    name = ndb.StringProperty()
+    value = ndb.FloatProperty()
+
+class userMorphology(ndb.Model):
+    emailID = ndb.StringProperty()
+    mPairs = ndb.StructuredProperty(measurePair, repeated=True)
```

Comment:
I used Measurement (singular) since the class refers to an individual name/value pair.